src/components/Login/JellyfinLogin.tsx

```diff
@@ -125,7 +126,9 @@ const JellyfinLogin: React.FC<JellyfinLoginProps> = ({
                         id="username"
                         name="username"
                         type="text"
-                        placeholder={intl.formatMessage(messages.username)}
+                        placeholder={`${intl.formatMessage(
+                          messages.email
+                        )} / ${intl.formatMessage(messages.username)}`}
```

Comment:
Leaving only "username" in this placeholder for Jellyfin users so they don't get confused:
```suggestion
                        placeholder={
                          settings.currentSettings.mediaServerType ===
                          MediaServerType.JELLYFIN
                            ? intl.formatMessage(messages.username)
                            : `${intl.formatMessage(
                                messages.email
                              )} / ${intl.formatMessage(messages.username)}`
                        }
```